File: sub-mysql-web-service/src/main/java/org/github/mysql/web/base/service/impl/TabSafeQuestionTypeServiceImpl.java
package org.github.mysql.web.base.service.impl;

import org.github.mysql.web.base.entity.TabSafeQuestionTypeEntity;
import org.github.mysql.web.base.mapper.ITabSafeQuestionTypeMapper;
import org.github.mysql.web.base.service.ITabSafeQuestionTypeService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 安全问题类别 服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-21
 */
@Service
public class TabSafeQuestionTypeServiceImpl extends ServiceImpl<ITabSafeQuestionTypeMapper, TabSafeQuestionTypeEntity> implements ITabSafeQuestionTypeService {}

File: sub-mysql-web-service/src/main/java/org/github/mysql/web/base/service/impl/TabSafeQuestionTypeServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.web.base.service.impl;

import org.github.mysql.web.base.entity.TabSafeQuestionTypeEntity;
import org.github.mysql.web.base.mapper.ITabSafeQuestionTypeMapper;
import org.github.mysql.web.base.service.ITabSafeQuestionTypeService;
import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 安全问题类别 服务实现类
 *
 * @author JYD_XL
 * @since 2021-05-31
 */
@Service
open class TabSafeQuestionTypeServiceImpl : ServiceImpl<ITabSafeQuestionTypeMapper, TabSafeQuestionTypeEntity>(), ITabSafeQuestionTypeService
